openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestSchemas.java

```diff
@@ -202,4 +202,28 @@
             0,
             TestSchemas.NUMERICALS_SCHEMA_WITH_LABEL_AT_END.getPredictiveFields()
     );
+
+    /**
+     * Schema with a feature with non-ASCII characters.
+     */
+    static final DatasetSchema SCHEMA_WITH_ONE_NON_ASCII_FEATURE = new DatasetSchema(
+            0,
+            ImmutableList.<FieldSchema>builder()
+                    .addAll(NUMERICALS_SCHEMA_WITH_TARGET_AT_START.getFieldSchemas())
+                    .add(new FieldSchema("cão", 5,
+                            new NumericValueSchema(false)))
+                    .build()
+    );
+
+    /**
+     * Schema with two feature with non-ASCII characters.
+     */
+    static final DatasetSchema SCHEMA_WITH_TWO_NON_ASCII_FEATURES = new DatasetSchema(
```

Comment:
`SCHEMA_WITH_TWO_NON_ASCII_FEATURES`
If you change the javadoc you should change it to `SCHEMA_WITH_TWO_NON_ASCII_CHARACTERS` instead of `SCHEMA_WITH_TWO_NON_ASCII_FEATURES`